starter/spring-boot-starter-parent/build.gradle

```diff
@@ -298,6 +298,16 @@ publishing.publications.withType(MavenPublication) {
 									configuration {
 										delegate.classesDirectory('${project.build.outputDirectory}')
 										delegate.requiredVersion('22.3')
+										exclusions {
+											exclusion {
+												delegate.groupId('org.springframework.boot')
+												delegate.artifactId('spring-boot-devtools')
+											}
+											exclusion {
```

Comment:
What is this? While I think this would be eventually useful, this feels like separate isn't it?